recursive_count_th/count_th.py

```diff
@@ -1,10 +1,22 @@
 '''
 Your function should take in a single parameter (a string `word`)
-Your function should return a count of how many occurences of ***"th"*** occur within `word`. Case matters.
+Your function should return a count of how many occurrences of ***"th"*** occur within `word`. Case matters.
 Your function must utilize recursion. It cannot contain any loops.
 '''
+
+
 def count_th(word):
-    
-    # TBC
-    
-    pass
+    # # An iterative solution to help conceptualize its corresponding recursive solution:
+    # count = 0
+    # for i in range(0, len(word) - 1):
+    #     if word[i] == 't' and word[i + 1] == 'h':
+    #         count += 1
+    # return count
+
```

Comment:
you could combine line 19 into word[0:2] == 'th'
 you could then call recursive_counter(word[2:], count)
else:
   recurcive_counter(word[1:], count)

Reply:
Great suggestion, thanks!